docs/kratos/social-signin/65_netid.mdx

```diff
@@ -6,10 +6,68 @@ sidebar_label: NetID
 
 # NetID
 
+````mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add NetID as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the NetID logo to start the configuration.
+3. Copy the Redirect URI and save it for later use.
+4. Create a [NetID Service](https://developerzone.netid.dev/latest/devportal/tutorial/services/) and
+   [NetID Client](https://developerzone.netid.dev/latest/devportal/tutorial/clients/).
+5. Enter the Redirect URI you copied from Ory to the **Callback URL** field of the NetID Client.
+6. After creating the NetID Client, the client is initially in _sandbox mode_, which is a pre-production state
+   prior to the release of your NetID Service.
+   If you want to test your NetID Client while it is in sandbox mode, you need to create one or more test users,
+   by going to the NetID Service page and clicking **Add test user**.
+7. Copy the following data from the NetID Client settings to the corresponding fields in the form on the Ory Console:
+    - **Client ID**
+    - **Client secret** (sandbox or live, depending on the status of the NetID Service)
+8. In the **Scopes** field of the form on the Ory Console, add the following scopes:
+    - `openid`
+    - `email`
+9. In the **Data Mapping** field of the form on the Ory Console, add the following Jsonnet code snippet:
```

Comment:
I commented on this already but it seems you didn't get me. 
In the docs you edited, please provide an explanation of what the Jsonnet does.

You can use a simple sentence like the one that's present in the CLI variant in many cases and say where it must be added in the Console:

> Create a Jsonnet code snippet to map the desired claims to the Ory Identity schema.

Reply:
Added the text ", which maps the desired claims to the Ory Identity schema"